### PR TITLE
adding Car Rental category page

### PR DESCRIPTION
**Pull request by kadarikalyani**

### CODE REVIEW

File: application/controllers/CarRentals.php [hunk 1]
```diff
@@ -22,9 +22,9 @@ public function __construct()
         $this->pageNameSingular         = 'CarRental';
         $this->pageView                 = 'carRentalsView';
         $this->hideSearchForm           = true;
-        $this->pageKeywords .= 'car rentals, National car hire, National car rental comparison , National car rentals , National carhire , National carrental , National carrentals , National cheap car hire , National cheap car rental , National cheap car rentals , National compare car rental , National rent a car , National rent car   , National rental car , National rental cars, National rentalcars , rent a car, rental car, rental cars,car rental,National car rental,';
-        $this->websiteDescriptionsTitle = 'Top Boat Websites To Buy a Used or New Boat';
-        $this->websiteDescriptionsIntro = 'There are tons of cars that are for rental in many local/national boat sites. These Top Boat websites make purchasing it fun and easy too.';
+        $this->pageKeywords .= 'car rentals,car hire , car rental comparison ,carrentals ,cheap car hire ,cheap car rentals ,compare car rental ,National rentalcars , rent a car,';
```
Comment (abhinaykrupa) [inline application/controllers/CarRentals.php]: remove space between hire & ,(comma)

Comment (abhinaykrupa) [inline application/controllers/CarRentals.php]: and also remove , at the very end of this statement

File: application/controllers/CarRentals.php [hunk 2]
```diff
@@ -22,9 +22,9 @@ public function __construct()
...
+        $this->websiteDescriptionsTitle = 'Top CarRental Websites To Rent a Car ';
```
Comment (abhinaykrupa) [inline application/controllers/CarRentals.php]: give space between Car & Rental

Comment (abhinaykrupa) [inline application/controllers/CarRentals.php]: remove space after 'Rent a Car'

File: application/controllers/CarRentals.php [hunk 3]
```diff
@@ -22,9 +22,9 @@ public function __construct()
...
+        $this->websiteDescriptionsIntro = 'There are tons of cars that are for rental in many local/national carRental sites. These Top CarRental websites Rent a car it fun and easy too.';
```
Comment (abhinaykrupa) [inline application/controllers/CarRentals.php]: endhe ee sentence ... its not making sense to read. change it to something that makes sense and dont copy it as is from other category page .. i told you to change it take make it unique

Comment (abhinaykrupa) [inline application/controllers/CarRentals.php]: also gap(space) between Car & Rental

File: application/core/MY_Controller.php [hunk 4]
```diff
@@ -40,6 +40,7 @@ class MY_Controller extends CI_Controller
         'motorcycles' => '<b>MOTORCYCLES</b> - Best Online Motorcycle Directory Websites To Find a Motorcycle',
         'boats' => '<b>BOATS</b> - Top Boat Websites To Buy a Used or New Boat',
         'furnitures'=> '<b>FURNITURES</b>-Top Websites To Shop For Furniture Online',
+        'carRentals'=> '<b>CARRENTALS</b>-Top CarRental Websites To Rent a Car online',
```
Comment (abhinaykrupa) [inline application/core/MY_Controller.php]: space between car & rentals in all the carrentals word occurance

**Comment by kadarikalyani** [comment]
Removing all changes from Car Rental page

**Comment by kadarikalyani** [comment]
removing all changes from Car Rental page

File: application/controllers/CarRentals.php [hunk 5]
```diff
@@ -20,11 +20,11 @@ public function __construct()
         $this->pageId                   = 'carRentals';
         $this->pageName                 = 'CarRentals';
         $this->pageNameSingular         = 'CarRental';
-        $this->pageView                 = 'carRentalsView';
+        $this->pageView                 = 'car RentalsView';
```
Comment (abhinaykrupa) [inline application/controllers/CarRentals.php]: @kadarikalyani no space here becoz it is refering to the carRentalsView file in the views folder ... hope this makes sense !

File: application/controllers/CarRentals.php [hunk 6]
```diff
@@ -20,11 +20,11 @@ public function __construct()
...
         $this->hideSearchForm           = true;
-        $this->pageKeywords .= 'car rentals, National car hire, National car rental comparison , National car rentals , National carhire , National carrental , National carrentals , National cheap car hire , National cheap car rental , National cheap car rentals , National compare car rental , National rent a car , National rent car   , National rental car , National rental cars, National rentalcars , rent a car, rental car, rental cars,car rental,National car rental,';
-        $this->websiteDescriptionsTitle = 'Top Boat Websites To Buy a Used or New Boat';
-        $this->websiteDescriptionsIntro = 'There are tons of cars that are for rental in many local/national boat sites. These Top Boat websites make purchasing it fun and easy too.';
+        $this->pageKeywords .= 'car rentals, car hire, car rental comparison, carrentals, cheap car hire, cheap car rentals, compare car rental, National rentalcars, rent a car,';
+        $this->websiteDescriptionsTitle = 'Top Car Rental Websites To Rent a Car ';
+        $this->websiteDescriptionsIntro = '';
```
Comment (abhinaykrupa) [inline application/controllers/CarRentals.php]: @kadarikalyani intro is missing please come up with an intro by looking at other pages as an inspiration on what to write up there ... that content should be unique

File: application/core/MY_Controller.php [hunk 7]
```diff
@@ -40,6 +40,7 @@ class MY_Controller extends CI_Controller
         'motorcycles' => '<b>MOTORCYCLES</b> - Best Online Motorcycle Directory Websites To Find a Motorcycle',
         'boats' => '<b>BOATS</b> - Top Boat Websites To Buy a Used or New Boat',
         'furnitures'=> '<b>FURNITURES</b>-Top Websites To Shop For Furniture Online',
+        'car Rentals'=> '<b>CAR RENTALS</b>-Top Car Rental Websites To Rent a Car online',
```
Comment (abhinaykrupa) [inline application/core/MY_Controller.php]: @kadarikalyani  no space in the key of the array 
'car Rentals'=> 
it is ok to have the space in the key of an array in php but no keys will have space in our framework, thats a standard that we follow

File: application/core/MY_Controller.php [hunk 8]
```diff
@@ -40,6 +40,7 @@ class MY_Controller extends CI_Controller
         'motorcycles' => '<b>MOTORCYCLES</b> - Best Online Motorcycle Directory Websites To Find a Motorcycle',
         'boats' => '<b>BOATS</b> - Top Boat Websites To Buy a Used or New Boat',
         'furnitures'=> '<b>FURNITURES</b>-Top Websites To Shop For Furniture Online',
+        'carRentals'=> '<b>CARRENTALS</b>-Top Car Rental Websites To Rent a Car online',
```
Comment (abhinaykrupa) [inline application/core/MY_Controller.php]: @kadarikalyani please use your brain ... i said no space in the keys of the array ... but not in the <b>CARRENTALS</b>

File: application/controllers/CarRentals.php [hunk 9]
```diff
@@ -22,9 +22,9 @@ public function __construct()
...
         $this->hideSearchForm           = true;
-        $this->pageKeywords .= 'car rentals, National car hire, National car rental comparison , National car rentals , National carhire , National carrental , National carrentals , National cheap car hire , National cheap car rental , National cheap car rentals , National compare car rental , National rent a car , National rent car   , National rental car , National rental cars, National rentalcars , rent a car, rental car, rental cars,car rental,National car rental,';
-        $this->websiteDescriptionsTitle = 'Top Boat Websites To Buy a Used or New Boat';
-        $this->websiteDescriptionsIntro = 'There are tons of cars that are for rental in many local/national boat sites. These Top Boat websites make purchasing it fun and easy too.';
+        $this->pageKeywords .= 'car rentals, car hire, car rental comparison, carrentals, cheap car hire, cheap car rentals, compare car rental, National rentalcars, rent a car,';
+        $this->websiteDescriptionsTitle = 'Top Car Rental Websites To Rent a Car ';
+        $this->websiteDescriptionsIntro = 'The Best wibsites To Rent a Car online ';
```
Comment (abhinaykrupa) [inline application/controllers/CarRentals.php]: @kadarikalyani why do u need a empty space at the end of the statments in the above 2 variables?

Comment (abhinaykrupa) [inline application/controllers/CarRentals.php]: @kadarikalyani please remove that extra empty space at the end

**Pull request merged**